receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -53,6 +55,95 @@ func (s *brokerScraper) shutdown(context.Context) error {
 	return nil
 }
 
+func (s *brokerScraper) scrapeConsumerFetch(now pcommon.Timestamp, allMetrics saramaMetrics, brokerID int64) {
+	key := fmt.Sprint("consumer-fetch-rate-for-broker-", brokerID)
+
+	if metric, ok := allMetrics[key]; ok {
+		if v, ok := metric["mean.rate"].(float64); ok {
+			s.mb.RecordKafkaBrokersConsumerFetchRateDataPoint(now, v, brokerID)
+		}
+	}
+}
```

Comment:
These functions look near identical, would it be possible to abstraction so you don't need to repeat yourself as much?

Reply:
done